src/Gui/Components/Collapsible.luau

```diff
@@ -0,0 +1,63 @@
+local Hoarcekat = script:FindFirstAncestor("Hoarcekat")
+local React = require(Hoarcekat.Packages.React)
+
+local OFFSET = 8
+
+local Assets = require(Hoarcekat.Plugin.Assets)
+
+local e = React.createElement
+local FitComponent = require(Hoarcekat.Plugin.Gui.Components.FitComponent)
+local IconListItem = require(Hoarcekat.Plugin.Gui.Components.IconListItem)
+
+type Props = {
+	children: any?,
+	title: string,
+}
+
+function Collapsible(props: Props)
+	local open, setOpen = React.useState(true)
+
+	local content = open and props.children
+
+	return e(FitComponent, {
+		containerClass = "Frame",
+		containerProps = {
+			BackgroundTransparency = 1,
+		},
+		layoutClass = "UIListLayout",
+		layoutProps = {
+			Padding = UDim.new(0, 5),
+			SortOrder = Enum.SortOrder.LayoutOrder,
+		},
+	}, {
+		Topbar = e(IconListItem, {
+			activated = function()
+				setOpen(function(current)
+					return not current
+				end)
+			end,
+			icon = open and Assets.collapse_down or Assets.collapse_right,
+			text = props.title,
+		}),
+
+		Content = content and e(FitComponent, {
+			containerClass = "Frame",
+			containerProps = {
+				BackgroundTransparency = 1,
+				Position = UDim2.fromOffset(OFFSET, 0),
+			},
+			layoutClass = "UIListLayout",
+			layoutProps = {
+				Padding = UDim.new(0, 5),
+			},
+		}, {
+			UIPadding = e("UIPadding", {
+				PaddingLeft = UDim.new(0, OFFSET),
+			}),
+
+			Content = e(React.Fragment, nil, content),
```

Comment:
Instead of doing this, pass `content` in as the fourth member of createElement. Fragments shouldn't be used outside of return values or to store IMO.

Reply:
Discussed via Discord, will retain this approach instead of fourth argument